.github/labeler.yml

```diff
@@ -228,6 +228,9 @@ udev-rules:
 virtfs:
   - modules.d/95virtfs/*
 
+virtiofs:
```

Comment:
Nitpick: I'd split this into a separate commit, as it's an infrastructure thing, not a part of the feature.